.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -2,8 +2,6 @@ name: $(Date:yyyyMMdd)$(Rev:.r)_$(Build.SourceBranchName)_$(BuildID)
 trigger: none
 
 variables:
-  - name: CONTAINER_IMAGE
-    value: 'mcr.microsoft.com/oss/azcu/go-dev:v1.38.1'
```

Comment:
Got you that the go-dev container is deprecated so we removed it. Does it mean the build will now be running on the Agent host instead of the go-dev container on the Agent?

Reply:
The build will be running on the packer VM that is provisioned by the code/packer templates. This will be orchestrated by the agent VM. It used to be orchestrated on the agent vm in the container. This PR just removes the container from the process.